extension/src/paymentHandler/refund-payment.handler.js

```diff
@@ -0,0 +1,39 @@
+const Promise = require('bluebird')
+const pU = require('./payment-utils')
+const { refund } = require('../service/web-component-service')
+const {
+  CTP_INTERACTION_TYPE_REFUND,
+} = require('../config/constants')
+
+async function execute (paymentObject) {
+  const refundInitTransactions = pU.listRefundTransactionsInit(paymentObject)
```

Comment:
what if we do not find any refund transactions?

Reply:
If we don't find any refund transactions, then it is not a refund and this method will not be executed.